src/components/SiteMenuSider.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div
+    class="fixed top-0 left-0 z-50 w-full h-screen bg-black bg-opacity-[30%]"
+  >
+    <div
+      class="flex flex-col items-center gap-[22px] w-full bg-white px-[18px] py-[42px]"
+    >
+      <div
+        class="flex items-center justify-between w-full text-like-green mb-[40px]"
+      >
```

Comment:
Can we use header tag?

Reply:
[🎨 Rename mobile menu](https://github.com/likecoin/liker-land/pull/1724/commits/39177508273139c8d56229738b6ad1ea92e2b127)